guac-visualizer.md

```diff
@@ -35,24 +34,24 @@ that there may be some unexpected behavior or usage problems.
 - A fresh copy of the [GUAC service infrastructure through Docker
   Compose]({{ site.baseurl }}{%link setup.md %})
 
-## Step 1: Clone the GUAC visualizer repo
+## Step 1: Getting started
 
-1. We highly suggest cloning the GUAC visualizer repo in the same path as you
-   cloned the main GUAC repo:
+1. If you haven't already, download the main GUAC version `v0.1.2` from
+   [this link](https://github.com/guacsec/guac/releases/tag/v0.1.2).
+   - Follow steps 2-5 in https://docs.guac.sh/setup to get it running.
```

Comment:
To link to setup use something like [setup]({{ site.baseurl }}{%link setup.md %})

Reply:
@jeffmendoza done, let me know if this works.